src/data/actions/individualAssignment.js

```diff
@@ -0,0 +1,42 @@
+import {
+  INDIVIDUAL_ASSIGNMENT_REQUEST,
+  INDIVIDUAL_ASSIGNMENT_SUCCESS,
+  INDIVIDUAL_ASSIGNMENT_FAILURE,
+} from '../constants/individualAssignment';
+
+
+import EcommerceApiService from '../services/EcommerceApiService';
+
+const fetchIndividualAssignmentRequest = () => ({
+  type: INDIVIDUAL_ASSIGNMENT_REQUEST,
+});
+
+const fetchIndividualAssignmentSuccess = data => ({
+  type: INDIVIDUAL_ASSIGNMENT_SUCCESS,
+  payload: {
+    data,
+  },
+});
+
+const fetchIndividualAssignmentFailure = error => ({
+  type: INDIVIDUAL_ASSIGNMENT_FAILURE,
+  payload: {
+    error,
+  },
+});
+
+
+const fetchIndividualAssignment = options => (
+  (dispatch) => {
+    dispatch(fetchIndividualAssignmentRequest(options));
+    return EcommerceApiService.fetchIndividualAssignment(options)
```

Comment:
Right now, the `EcommerceApiService` has some placeholder functions for the other 2 API calls currently being used for the code management page. You could create an additional placeholder in there for this function.

As is, once the `CodeAssignmentModal` is wired up to these actions/reducers, this would likely throw a JS error because you're trying to invoke a function that doesn't exist yet.

Related to the overall PR comment, this function should also be named generic enough to be applicable for both individual and bulk assign.